test/test_functional.py

```diff
@@ -47,28 +47,28 @@ def test_below_ui(self):
         time.sleep(10)
         self.verify_functional_test()
 
-    @unittest.skipIf("TRAVIS" in os.environ and os.environ["TRAVIS"] == "true", "Skipping this test on Travis CI.")
-    def test_from_ui_new_text(self):
-        """Test function which injects the user input data at the presentation later
-        to check the end to end functionality"""
-        window = Tk()
-        win = TextSimilarityWindow(window)
-        win.check_is_new_text.invoke()
-        win.path_t.insert(0, str(TestResource.file_path))
-        win.uniq_id_t.insert(0, 0)
-        win.steps_t.insert(0, "1,2")
-        time.sleep(2)
-        win.new_text.insert(0, "a3 d4")
-        win.submit.invoke()
-        time.sleep(10)
-        window.quit()
-        self.verify_functional_test(True)
+    # @unittest.skipIf("TRAVIS" in os.environ and os.environ["TRAVIS"] == "true", "Skipping this test on Travis CI.")
+    # def test_from_ui_new_text(self):
+    #     """Test function which injects the user input data at the presentation later
+    #     to check the end to end functionality"""
+    #     window = Tk()
+    #     win = TextSimilarityWindow(window)
+    #     win.check_is_new_text.invoke()
+    #     win.path_t.insert(0, str(TestResource.file_path))
+    #     win.uniq_id_t.insert(0, 0)
+    #     win.steps_t.insert(0, "1,2")
+    #     time.sleep(2)
+    #     win.new_text.insert(0, "a3 d4")
+    #     win.submit.invoke()
+    #     time.sleep(10)
+    #     window.quit()
+    #     self.verify_functional_test(True)
```

Comment:
Should we remove the commented code?

Reply:
They are UI test which can be done at desk, but CI it is not possible hence kept as commented. In travis there was a skip option. such option i am not able to get in github action. Hence keeping it commented untill i find a way to skip the test in CI